### ISSUE TITLE
Put space between consecutive checkboxes

### ISSUE THREAD

**Issue by nicogarcia** [nicogarcia/classes-web-client#104]
Están en "/signup/teacher/data?step=2"

**Comment by nicogarcia** [comment]
También para radio buttons.

**Comment by federicojasson** [comment]
¿Lo tengo que hacer yo?

**Comment by nicogarcia** [comment]
Me habías dicho que sí, es viejo esto. Igual es muy poco prioritario. No te hagas drama todavía.

**Comment by federicojasson** [comment]
Yo había entendido que la idea era separarlos en dos columnas. Lo otro no me acuerdo.
Cuanto antes cerremos issues, más claro va a estar el panorama.

**Comment by nicogarcia** [comment]
No, eso es otro issue, este es de los checkboxes de Grupal Individual y los de horarios, es un problema de css, quedan muy pegados.
